tests/pytests/test.py

```diff
@@ -2457,17 +2457,6 @@ def testOptionalFilter(env):
 
     r = env.cmd('ft.search', 'idx', '~(word20 => {$weight: 2.0})')
 
-def testCriteriaTesterDeactivated():
```

Comment:
Just out of curiosity, what was the purpose of this test?

Reply:
@MeirShpilraien

Reply:
Making sure this feature is disabled by default. It was made for graph, but they ended up not using it.